File: kinotic-domain/src/main/java/org/kinotic/domain/api/model/OrganizationStorageStatusType.java
package org.kinotic.domain.api.model;

/**
 * The lifecycle states of an organization's storage, carried by its
 * {@link OrganizationStorageStatus}.
 */
public enum OrganizationStorageStatusType {

    /**
     * The storage account and its private endpoint are being created; a deployment that
     * needs the storage waits for it.
     */
    PROVISIONING,

    /**
     * The storage exists and the organization's deployments can publish to it.
     */
    READY,

    /**
     * Provisioning did not complete; the status message says why. The next deployment that
     * needs the storage tries again.
     */
    FAILED
}

File: kinotic-js/workspace/packages/management-api/src/api/model/OrganizationStorageStatusType.ts
/**
 * The lifecycle states of an organization's storage, carried by its OrganizationStorageStatus.
 */
export enum OrganizationStorageStatusType {
    /**
     * The storage account and what the platform reaches it through are being created.
     */
    PROVISIONING = 'PROVISIONING',
    /**
     * The storage is usable: deployments publish to it.
     */
    READY = 'READY',
    /**
     * Provisioning failed; the status message says why, and it can be run again.
     */
    FAILED = 'FAILED'
}
